### ISSUE TITLE
Blockly Script Editing Window 2-Finger Navigation with Touchpad

### ISSUE THREAD

**Issue by AdamMcAdamson** [Adam13531/BotLand#33]
**Steps to reproduce**
Using 2 fingers on touchpad in Blockly Script editor.

**Expected outcome**
have code window scroll down or side-to-side

**Actual outcome**
code window zooms in and out.

**Bowser information**
{"name":"Chrome","chrome":true,"version":"55.0","blink":true,"windows":true,"a":true}

**In-game information**
Player ID: 3768
Player name: AdamMcAdamson

**Comment by HiDeoo** [comment]
The behaviour is the same when using a mouse, the scroll wheel is used to zoom-in/out. Do you get something different when using a mouse?

**Comment by AdamMcAdamson** [comment]
No, a mouse acts the same. I guess this is more of a design suggestion than a bug.

The zooming is hyper-sensitive making it unusable on a touch pad. Also, the directions for zooming aren't user friendly and seem inconsistent (moving left/right cause repeating zoom-in and zoom-out even if the direction doesn't change).

I suggest having the 2-finger touchpad be used for window movement instead.
Even if not, the current purpose isn't served by it's implementation.

**Comment by HiDeoo** [comment]
Ho ok, gotcha. Thanks for clarifying.

Regarding the bug only, this should be fixed by [blockly/#758](https://github.com/google/blockly/issues/758) in a more recent version of Blockly than the one @Adam13531  is currently using.

**Comment by Skedog** [comment]
As an update to this, it appears the Blockly version being used has been updated to a version where this was fixed, as this issue is no longer present in Bot Land. Using two fingers on a touchpad is a very smooth experience now. Leaving this open as it is still an enhancement suggestion.
